File: f.py
s1 = int(input("Enter length of side 1 : "))
s2 = int(input("Enter length of side 2 : "))
s3 = int(input("Enter length of side 3 : "))
s  = float((s1+s2+s3)/2)

print("Area is %f" % (s*(s-s1)*(s-s2)*(s-s3))**(float(1/2)) )

File: let-us-c/chapter2/f.c
#include<stdio.h>
#include<math.h>

int main(){
	int a, b, c;
	a= b = c  =0;
	float s, res;

	printf("Enter Sides of Triangle\n");
	printf("Side 1: ");
	scanf("%d",&a);
	printf("Side 2: ");
	scanf("%d",&b);
	printf("Side 3: ");
	scanf("%d",&c);

	s = (a+b+c)/2;
	res = sqrt(s*(s-a)*(s-b)*(s-c));
	printf("Area is : %f \n", s );
	
	return 0;
}